src/coreclr/vm/ClrEtwAll.man

```diff
@@ -3313,6 +3332,10 @@
                            keywords ="ThreadingKeyword"  opcode="Wait"
                            task="ThreadPoolWorkerThread"
                            symbol="ThreadPoolWorkerThreadWait" message="$(string.RuntimePublisher.ThreadPoolWorkerThreadEventMessage)"/>
+                  
+                    <event value="58" version="0" level="win:Informational" template="YieldProcessorMeasurement"
```

Comment:
I think this deserves a new keyword so that it can be captured without opting into other threading events.  This will allow these values to be captured at scale without having an impact on overhead when threading events are not desired.  Maybe we make the keyword a "configuration" keyword so that other low verbosity events that are considered configuration can be emitted here as well without having to create a new keyword for each category of configuration.

Reply:
Isn't it possible to enable just one event under a keyword with enabling all events under the keyword? It seems to be possible with PerfView by specifying the event ID in the providers switch.

Reply:
Edited, I meant "without enabling all events under the keyword"

Reply:
Yeah, you're right.  I had not seen that before.  I looked, and this is plumbed all the way through to ETW, so it should be fine.  EventPipe won't allow this, but it certainly could be implemented if necessary.